dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -739,4 +762,29 @@ public TopicSelection getTopicSelection() {
     public void setTopicSelection(TopicSelection topicSelection) {
         this.topicSelection = topicSelection;
     }
+
+    public Set<Author> getAuthors() {
+        T realDefaultVersion = this.getActualDefaultVersion();
```

Comment:
This implementation is much better that the duplication we had before.

Reply:
Yes, this is good.  Ideally, IMHO, very-little-to-nothing is stored in `Entry` (and its derivatives) that can change when a new version is created.